rpc/common/types.go

```diff
@@ -13,9 +13,8 @@ import (
 // BlockArgs is struct to include optional block formatting params.
 type BlockArgs struct {
 	WithSigners bool     `json:"withSigners"`
-	InclTx      bool     `json:"inclTx"`
 	FullTx      bool     `json:"fullTx"`
-	Signers     []string `json:"signers"`
+	Signers     []string `json:"-"`
```

Comment:
why change to -?

Reply:
The field is just a field to store in-memory computation result, and is not used in RPC calls. Thus I disabled the json serialization of this field. 

Reference at https://api.hmny.io/#52f8a4ce-d357-46f1-83fd-d100989a8243.